### ISSUE TITLE
Missing some includes during complinig

### ISSUE THREAD

**Issue by PaulKobaCW** [ConnorKirkpatrick/E220Lib#4]
Hello Connor,
Im facing several errors during compilin sketch

```
In file included from C:\Users\admin\AppData\Local\Temp\.arduinoIDE-unsaved2023820-14888-vmu0vk.eabwm\Receive\Receive.ino:2:0:
c:\Users\admin\Documents\Arduino\libraries\E220Lib\src/E220.h:9:10: fatal error: USB/USBAPI.h: No such file or directory
 #include "USB/USBAPI.h"
          ^~~~~~~~~~~~~~
compilation terminated.

exit status 1

Compilation error: exit status 1
```

its reproducing only on 1.3 version of e220lib form offcial arduino repo

on 1.1.2 i have problems with pins m0, if i set it how it should be defined module.init returns

```
17:33:40.216 -> Error reading module config, check the wiring
17:33:40.249 -> Issue initiating the module
```
if i replace it mo --> aux, aux --> mo, init gone fine but i suppose that module works in wrong mode because wrong voltage levels on  pins/
i got stuck(

**Comment by ConnorKirkpatrick** [comment]
Thanks for the heads up, I'll look into it this weekend. Initial impression is that it's a bad import that made it into release while I was tinkering with support for more boards

**Comment by paulignari** [comment]
I encountered the same error using the library provided by Plaformio (1.3.0).

The library from the repository (branch master, commit f177b41) compiles fine.

Maybe you need to push the latest version (1.3.2) in the Arduino IDE and Platformio library collections.

**Comment by PaulKobaCW** [comment]
> Thanks for the heads up, I'll look into it this weekend. Initial impression is that it's a bad import that made it into release while I was tinkering with support for more boards

Thnaks a lot,
btw i figured out with m0 pin  - it was my mistake in wiring, so this one may skip.

**Comment by ConnorKirkpatrick** [comment]
Hi, this should be fixed in 1.3.3 which should be in the library manager in the next few hours. I don't have my module handy to check so if this works please let me know :)

**Comment by paulignari** [comment]
Now version 1.3.3 is in Platformio Library Registry and compiles well, thanks!
